src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -516,26 +516,37 @@ public static function parseSimpleTokens($strString, $arrData)
 	{
 		$strReturn = '';
 
-		// Remove any unwanted tags (especially PHP tags)
-		$strString = strip_tags($strString, \Config::get('allowedTags'));
+		// Remove PHP code from string. Not supported for security reasons.
+		$strString = preg_replace('/<script language="php">.*<\/script>/', '', $strString);
+		$strString = preg_replace('/<(\?|\%)\=?(php)?.*(\%|\?)>/', '', $strString);
```

Comment:
Then we should go for @ausi's suggestion and throw an exception if we detect a PHP tag, shouldn't we?